jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
```

Comment:
@pavolloffay Jenkins also running on the same namespace, So I guess we may not use delete namespace here.